docs/getting_started.rst

```diff
@@ -0,0 +1,49 @@
+===============
+Getting Started
+===============
+
+:doc:`Wagtail Autocomplete <index>` provides an edit handler that allows an editor to
+select related objects via a quick autocompleted searching interface.
+
+Requirements
+============
+
+Wagtail Autocomplete requires Wagtail version 1.11 or newer.
+
+.. warning::
+
+    We do not currently support Wagtail 2.x, but we're working on it!
+
+Installation
+============
+
+Install with ``pip``:
+
+.. code-block:: sh
+
+    pip install https://github.com/emilyhorsman/wagtail-autocomplete/tarball/master/
+
+Setup
+=====
+
+Add ``'wagtailautocomplete'`` to your project's ``INSTALLED_APPS``. This enables Wagtail to auto-discover the Wagtail Autocomplete's admin hooks.
+
+Add Wagtail Autocomplete's url patterns to your project's url config, usually in ``urls.py``. This should still come before your ``wagtail_urls``:
```

Comment:
the Wagtail Autocomplete's -> Wagtail Autocomplete's
url patterns -> URL patterns
url config -> URL config
should still come -> should come